### PR TITLE
Create catch-all-addresses.md

### PR DESCRIPTION
**Pull request by hikarutilmitt**
will provide screenshots that fit into article

### CODE REVIEW

**Comment by nimbinatus** [comment]
Opened PR on the patch-14 branch, hikarutilmitt/rackspace-how-to#8 (note: I removed the screenshots to avoid issues with UI updates per how-to guidelines. If you think the screenshots are crucial to understanding how to follow the article's steps, we can certainly add them back in. Thanks!)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ea7f954f4d/how-to/). :bow:

File: content/cloud-sites/catch-all-addresses.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+A catch-all address receives all email sent to invalid mailboxes at a user's domain and is sometimes used to avoid missing any emails to a domain.
+
+**Warning:** Catch-all addresses are not filtered for spam. Use at your own risk. With this feature enabled, you will receive thousands of spam emails from spammers who guess random addresses within your domain.
+
+1. Log in to the Cloud Sites Email Control Panel.
+
+1. Click the **Domains** section.
+
+  ![Domains section of the Cloud Sites Email Control Panel](_assets/img/cloud-sites/catch-all-addresses/catchall1.png)
```
Comment (DonNateR) [inline content/cloud-sites/catch-all-addresses.md]: These type of markdown image directives don't work with Nexus (strange, I know). Use this format for image directives instead, ``

File: content/cloud-sites/catch-all-addresses.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+permalink: catch-all-addresses/
```
Comment (DonNateR) [inline content/cloud-sites/catch-all-addresses.md]: Permalink needs to reflect the article title. This is true with the file name as well, so `set-up-catch-all-address/` for the permalink, then `set-up-catch-all-addresses.md` for the article name.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7439a03d68/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c9cc5af32f/how-to/)

**Comment by nimbinatus** [comment]
@DonNateR , can you double-check me again?

**Pull request merged**